distributed/utils.py

```diff
@@ -1413,20 +1413,23 @@ class Log(str):
     """ A container for logs """
 
     def _repr_html_(self):
-        return "<pre><code>{log}</code></pre>".format(log=self)
+        if "\n" not in self:
+            return "<pre><code>{log}</code></pre>".format(log=self)
```

Comment:
I would be surprised if this is ever triggered. Most outputs will have a trailing new line and will likely be multiple lines. `Log` might not be the most appropriate name for this class as it will likely represent all logs from a single worker, scheduler, etc.

It might be better to just always add a leading line break and possibly add a trailing one if one is missing.